reference/user_defined_functions.rst

```diff
@@ -7,12 +7,83 @@ This section contains reference information for the User Defined Functions provi
 
 Table and Shard DDL
 -------------------
+.. _create_distributed_table:
+
+create_distributed_table
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+The create_distributed_table() function is used to define a distributed table
```

Comment:
Could we use distributed in place of partitioned? Currently, it uses a mix of both partitioned and distributed and we use distributed in most of our docs.